lib/kitchen/driver/hyperv.rb

```diff
@@ -187,6 +188,10 @@ def vhd_folder?
         config[:parent_vhd_folder] && Dir.exist?(config[:parent_vhd_folder])
       end
 
+      def enable_guest_services
+        @enable_guest_services ||= config[:enable_guest_services]

```

Comment:
I would leave out this method and just refer to this as `config[:enable_guest_services]`